package.json

```diff
@@ -40,5 +40,9 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "types": "./index.d.ts",
+  "dependencies": {
+    "@types/jquery": "^2.0.46"
```

Comment:
@jregistr I want to ask if this `dependencies` is necessary? I don't use typescript and use `npm` to install `bootstrap-table` will download this dependencies every time?